rclcpp/CHANGELOG.rst

```diff
@@ -2,6 +2,29 @@
 Changelog for package rclcpp
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+6.4.0 (2021-03-18)
+------------------
+* Document design decisions that were made for statically typed parameters (`#1568 <https://github.com/ros2/rclcpp/issues/1568>`_)
+* Fix doc typo in CallbackGroup constructor (`#1582 <https://github.com/ros2/rclcpp/issues/1582>`_)
+* Enable qos parameter overrides for the /parameter_events topic  (`#1532 <https://github.com/ros2/rclcpp/issues/1532>`_)
+* Add support for rmw_connextdds (`#1574 <https://github.com/ros2/rclcpp/issues/1574>`_)
+* Remove 'struct' from the rcl_time_jump_t. (`#1577 <https://github.com/ros2/rclcpp/issues/1577>`_)
+* Re-apply `#829 <https://github.com/ros2/rclcpp/issues/829>`_: Add ParameterEventsSubscriber class (`#1573 <https://github.com/ros2/rclcpp/issues/1573>`_)
```

Comment:
I typically merge an "add, revert, reapply" sequence in one item, the fact that was temporarily reverted doesn't seem to be worth noting in the changelogs.